_posts/2019-05-26-owengamingboy.md

```diff
@@ -0,0 +1,79 @@
+---
+layout:     post
+title:      "Deconstructivist Pop: A Case Study"
+author:     Rahul Tirumalareddy
+date:       2019-05-26
+categories: Owengamingboy
+album_art:  assets/images/we_did_it_dumb.jpg
+---
+
+Macaroni and cheese. Peanut butter and jelly. Hide and seek. Cookies and milk.
+Research and development. Crime and Punishment. Surf and sand. Fish and chips.
+Cheeseburgers and fries. Mom and dad. Marriage and divorce. Weekdays and
+weekends. The soccer field and Ben & Jerry's. Mom and another guy. Milk and
+honey. Hiroshima and Nagasaki. Some things go together like peas in a pod. 
```

Comment:
You don't want your recruiter to see this one

Reply:
yes i do.

Reply:
it displays my wit